### DISCUSSION TITLE
New navigation feedback

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#60732] [Other Feature Feedback, Questions, & Ideas]
I tried the navigation bar beta version, tthe new navigation bar is really awesome.

But this area isn't necessary anymore.
This option can shift to slider menu or popup options when we click on our profile picture on top right corner.

![IMG_20230713_085421.jpg](https://github.com/community/community/assets/94877880/5e59385b-1b0b-40f3-809d-1abb11cf9acd)

**Comment by taki-on** [comment]
Thank you for contacting us! The issue is about mobile web. I will change the category so that the team behind the page would make a contact with you.

**Comment by NitinIsFit** [comment]
> Thank you for contacting us! The issue is about mobile web. I will change the category so that the team behind the page would make a contact with you.

Thanks for listening to my feedback, but there are few more like

![Screenshot_2023-07-14-13-54-09-952_com.android.chrome.jpg](https://github.com/community/community/assets/94877880/5eefaab5-a08f-4a50-b444-c3ce1d5f22ef)

- the search bar goes out of screen.
- teams and settings page for the organizations never got mobile friendly website UI
